### PR TITLE
Added Upcoming Rocket Launch Schedule | Web Scrapper

### LINKED ISSUE #181: Upcoming Rocket Launch Schedule | Web Scrapper
# Description

The script will scrape the Details of the upcoming Rocket Launch Schedule and return in a JSON

## Type of issue

- [x] Feature (New Script)
- [ ] Bug
- [ ] Documentation

## Checklist:

- [x] I have read the project guidelines.
- [x] I have checked all the existing projects, before submitting a new project issue.
- [x] I have checked previous issues to avoid duplicates.
- [x] This issue will be meaningful for the project.

### PR DESCRIPTION
**Pull request by Rajdeep-Ray**
# Description

This script scrapes the upcoming Rocket Launch Schedule and saves it into a JSON and CSV File

Fixes #181 

## Type of change

Choosing one or more options from the following as per the nature of your Pull request.
- NOTE: *These boxes can be checked using **`[X]`***

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Documentation Update

# Checklist:
Please tick all the boxes that are fulfilled by your Pull Request.

- [x] I have named my files and folder, according to this project's guidelines.
- [x] My code follows the style guidelines of this project.
- [x] My Pull Request has a descriptive title. (not a vague title like `Update index.md`)
- [x] I have commented on my code, particularly in hard-to-understand areas.
- [x] I have created a helpful and easy to understand `README.md`, according to the given [`README_TEMPLATE.`](https://github.com/Python-World/Python_and_the_Web/blob/master/README_TEMPLATE.md)
- [x] I have included a requirements.txt file (if external libraries are required.)
- [x] My changes do not produce any warnings.
- [x] I have added a working sample/screenshot of the script.

### CODE REVIEW

File: Scripts/Web_Scrappers/Rocket-Schedule/requirements.txt [hunk 1]
```diff
@@ -0,0 +1,2 @@
+bs4
```
Comment (chavarera) [inline Scripts/Web_Scrappers/Rocket-Schedule/requirements.txt:1]: add specific version
use pip freeze
```
pip freeze list>requirements.txt
```

**Review by chavarera — CHANGES_REQUESTED** [review]

**Review by chavarera — CHANGES_REQUESTED** [review]
also solve py-linter error

**Comment by Rajdeep-Ray** [comment]
@chavarera the required changes have been done

**Review by chavarera — APPROVED** [review]

**Review by AdityaJ7 — APPROVED** [review]

**Comment by AdityaJ7** [comment]
@all-contributors please add @Rajdeep-Ray for code

**Pull request merged**